tools/scylla-nodetool.cc

```diff
@@ -1578,6 +1624,58 @@ void stop_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
     client.post("/compaction_manager/stop_compaction", {{"type", compaction_type}});
 }
 
+void tablehistograms_operation(scylla_rest_client& client, const bpo::variables_map& vm) {
+    const auto& [keyspace, table] = parse_keyspace_and_table(client, vm, "table");
```

Comment:
i'd suggest capture the return value by value instead of by reference. otherwise `keyspace` and `table` would be referencing two members of a temporary variable, which would be destructed after this expression is evaluated.

Reply:
done in v2